runtime/doc/options.txt

```diff
@@ -1515,6 +1515,20 @@ A jump table for the options with a short description can be found at |Q_op|.
 		    select one from the menu. Only works in combination with
 		    "menu" or "menuone".
 
+						*'completepopup'* *'cpp'*
+'completepopup' 'cpp'	string	(default "")
+			global
+		This option is used to set the completion window and info floating
+		preview window. The option is a comma-separated list of values:
+		`align`, `border`, `title`, `title_pos`, `footer`, `footer_pos`.
+		see |nvim_open_win|.
```

Comment:
`lua/help_spec.lua` complains here. Can't link to `nvim_open_win`?

Reply:
Needs `()`, change to `|nvim_open_win()|`.

Reply:
cool thx